### ISSUE TITLE
destroy.t fails

### ISSUE THREAD

**Issue by eserte** [ap/Async#1]
destroy.t fails on all of my smoker systems:
```
# Looks like you planned 26 tests but ran 16.
t/destroy.t .............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 10/26 subtests 
```

**Comment by ap** [comment]
Is there any more output than that…?

**Comment by ap** [comment]
Oh, I see.

**Comment by ap** [comment]
Fixed in 0.12
